### PR TITLE
Added Resources for Angular , Intro to CS , Intro to Data Science , Hosting and more

### PR DESCRIPTION
**Pull request by amitkrishna**
Hi @shahednasser  Please review this :
Curated list of resources for  Data Science , Intor to Computer Science , AngularJS , Hosting  and more.

### CODE REVIEW

**Comment by amitkrishna** [comment]
Can this PR be in for Hackoctoberfest 2020

**Comment by shahednasser** [comment]
I'll merge this in October @amitkrishna for hacktoberfest. Thank you.

**Comment by amitkrishna** [comment]
Thank you @shahednasser

File: README.md [hunk 1]
```diff
@@ -45,10 +45,11 @@ To contribute, please check out our contribution guide [here](https://github.com
 
 ---
 
-##AngularJS
+## AngularJS
 
--[Angular Academy](https://www.angularacademy.ca/)
--[W3 Schools](https://www.w3schools.com/angular/)
+- [Angular Academy](https://www.angularacademy.ca/)
+
```
Comment (shahednasser) [inline README.md:52]: Please remove this extra space between items

File: README.md [hunk 2]
```diff
@@ -75,6 +90,15 @@ To contribute, please check out our contribution guide [here](https://github.com
 - [Magento 2 Creating A UI Form](https://www.mageplaza.com/devdocs/creat-a-ui-form-in-magento-2.html)
 - [Tips for Magento 2 Layouts](https://medium.com/@shahedn/tips-for-magento-2-layouts-d7e076812fa2)
 
+---
+## Fundamentals of  Computer Science
```
Comment (shahednasser) [inline README.md:94]: Add new line before and after title

File: README.md [hunk 3]
```diff
@@ -75,6 +90,15 @@ To contribute, please check out our contribution guide [here](https://github.com
...
+- [CS50](https://cs50.harvard.edu/college/2020/fall/)
+
+---
+
+## Introduction to Data Science
```
Comment (shahednasser) [inline README.md:99]: Add new line after title

File: README.md [hunk 4]
```diff
@@ -13,32 +13,46 @@ To contribute, please check out our contribution guide [here](https://github.com
...
+- [AngularJS](#angularjs)
 - [Android](#android)
-- [C Sharp](#c-sharp)
 - [Others](#others)
   - [Cheat Sheets](#cheat-sheets)
   - [Design](#design)
+  - [Hosting](#Hosting)
   
 ---
+## HTML
```
Comment (shahednasser) [inline README.md:38]: Add new line before title

**Review by shahednasser — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (amitkrishna) [reply README.md:104]: Resolved

File: README.md (hunk 2 above)
Comment (amitkrishna) [reply README.md:98]: resolved

File: README.md (hunk 1 above)
Comment (amitkrishna) [reply README.md]: resolved

File: README.md [hunk 5]
```diff
@@ -13,32 +13,52 @@ To contribute, please check out our contribution guide [here](https://github.com
...
-  - [Data Structures](#data-structures)
-  - [Dynamic Programming](#dynamic-programming)
-  - [String Processing](#string-processing)
-- [Python](#python)
-- [Android](#android)
-- [C Sharp](#c-sharp)
-- [Others](#others)
-  - [Cheat Sheets](#cheat-sheets)
-  - [Design](#design)
+- [Awesome Resources ![Awesome](https://github.com/shahednasser/awesome-resources)](#awesome-resources-img-srchttpscdnrawgitcomsindresorhusawesomed7305f38d29fed78fa85652e3a63e154dd8e8829mediabadgesvg-altawesome)
```
Comment (amitkrishna) [inline README.md:16]: This Link can be removed Not sure what it is @shahednasser

File: README.md [hunk 6]
```diff
@@ -13,32 +13,52 @@ To contribute, please check out our contribution guide [here](https://github.com
...
+- [Purpose](#purpose)
+- [Contribution](#contribution)
```
Comment (amitkrishna) [inline README.md:18]: Can Purpose and Contribution be removed since this is already available in the beginning

File: README.md (hunk 5 above)
Comment (shahednasser) [reply README.md:16]: I think you added this by mistake. It is not in this repository.

File: README.md (hunk 6 above)
Comment (shahednasser) [reply README.md:18]: Those aren't in this repository either. Please remove them. We just need the items under Content

File: README.md [hunk 7]
```diff
@@ -13,32 +13,52 @@ To contribute, please check out our contribution guide [here](https://github.com
...
+- [Content](#content)
```
Comment (shahednasser) [inline README.md:19]: Remove Content and just keep the items under it.

**Pull request closed without merging**